gui/launcher.cpp

```diff
@@ -979,12 +982,12 @@ void LauncherSimple::build() {
 	_loadButton = loadButton;
 
 	// Add edit button
-	if (g_system->getOverlayWidth() > 320) {
+	if (g_system->getOverlayWidth() > 410) {
```

Comment:
Why?

Reply:
that's around the point where Game Options was too long

Reply:
I don't like this particular change. There is a consistent understanding currently that lowres is for width of 320 pixels or less. With that change it becomes inconsistent. Also that is only based on the English labels and may not necessarily play well for the translated ones.
We could consider changing the definition of lowres consistently everywhere (for example anything below 640 in width).

Reply:
I put it back to 320